packages/itwinui-css/backstop/tests/header.html

```diff
@@ -14,17 +14,18 @@
       content="width=device-width, initial-scale=1"
     >
     <title>Header | iTwinUI</title>
-    <link
-      rel="stylesheet"
-      href="../../css/all.css"
-    />
-    <link
-      rel="stylesheet"
-      href="../assets/demo.css"
-    />
+
+    <script type="module">
+      import "./assets/theme.js";
+      import "./assets/userIcon.js";
```

Comment:
I am running into an issue with backstop tests. Because this is an ESM import, it requires a dev server to be able to resolve. Simply opening the file (e.g. visiting the `file:///` path in browser) does not work.

@bentleyvk Do you know if there is an easy way to make backstop tests run on a server rather than on hardcoded html paths?

Reply:
Try chaning `scenario.url` in `packages\itwinui-css\backstop\scenarios\index.js`
![image](https://user-images.githubusercontent.com/36186912/179451195-370f2cf3-ef22-4605-a229-0d35f8ff4f0e.png)

Reply:
So I would need to start a server inside the docker container? Could use a custom entrypoint to do that, but then I won't have access to this `{backstopCommand}`

![ ](https://user-images.githubusercontent.com/9084735/179553342-0ae80e1a-3f1e-4a1a-8cc7-119762ac6b83.png)